src/shipit_dashboard/shipit_dashboard/models.py

```diff
@@ -32,7 +32,7 @@ class BugAnalysis(db.Model):
     parameters = sa.Column(sa.Text())
     created = sa.Column(sa.DateTime, default=datetime.datetime.utcnow)
 
-    bugs = db.relationship('BugResult', secondary=bugs, backref=db.backref('analysis', lazy='dynamic'))  # noqa
+    bugs = db.relationship('BugResult', secondary=bugs, backref=db.backref('analysis', lazy='dynamic'), order_by='BugResult.bugzilla_id')  # noqa
```

Comment:
i always wonder how to properly format this long lines of sqlalchemy queries. i still have no idea how to do them :)

to put it differently i dont really care how to style it if i can have a tool like go/elm has it to autoformat our code. at recent pycon in dublin this tool https://github.com/google/yapf was presented as a possible solution. i created a ticket to evaluate this as an option (#96)